src/components/MultiHopTrade/components/TradeInput/components/ConfirmSummary.tsx

```diff
@@ -89,6 +90,7 @@ export const ConfirmSummary = ({
   const buyAssetFeeAsset = useAppSelector(state =>
     selectFeeAssetById(state, buyAsset?.assetId ?? ''),
   )
+  const isAccountsLoading = useAppSelector(selectIsAccountsLoading)
```

Comment:
can add `isAccountsLoading` to `shouldDisablePreviewButton` for posterity/paranoia

Reply:
also nitpick `isLoadingAccounts` vernacular reads better imo

Reply:
Fixed in [e72bded](https://github.com/shapeshift/web/pull/7629/commits/e72bded769397fe997646cd23ed7ff15a20d7bc6)